{"name": "Invalid value for param - coords in the Request Body and getting Incorrect output -- @1.3 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Got Blank Response as the Output\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628093067466, "stop": 1628093067466}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628093067466, "stop": 1628093067467}, {"name": "When Set invalid request Body { \"roomSize\" : [5, 5], \"coords\" : [1], \"patches\" : [ [1, 0], [2, 2]], \"instructions\" : \"NNESEESWNWW\" }", "status": "passed", "start": 1628093067468, "stop": 1628093067468}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628093067468, "stop": 1628093067482}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Got Blank Response as the Output\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628093067483, "stop": 1628093067485}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5, 5], \"coords\" : [1], \"patches\" : [ [1, 0], [2, 2]], \"instructions\" : \"NNESEESWNWW\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628093067464, "stop": 1628093067487, "uuid": "0ede2815-1d20-4cc7-ac48-9114c2f35fae", "historyId": "9e484e8729912a0e580a7e7b29ee7328", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for coords params: Invalid value for param - coords in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for coords params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}